scipy/stats/_morestats.py

```diff
@@ -2732,18 +2737,103 @@ def levene(*samples, center='median', proportiontocut=0.05):
            Stanford University Press, pp. 278-292.
     .. [3] Brown, M. B. and Forsythe, A. B. (1974), Journal of the American
            Statistical Association, 69, 364-367
+    .. [4] C.I. BLISS (1952), The Statistics of Bioassay: With Special
+           Reference to the Vitamins, pp 499-503,
+           :doi:`10.1016/C2013-0-12584-6`.
+    .. [5] B. Phipson and G. K. Smyth. "Permutation P-values Should Never Be
+           Zero: Calculating Exact P-values When Permutations Are Randomly
+           Drawn." Statistical Applications in Genetics and Molecular Biology
+           9.1 (2010).
 
     Examples
     --------
-    Test whether or not the lists `a`, `b` and `c` come from populations
-    with equal variances.
+    In [4]_, the influence of vitamin C on the tooth growth of guinea pigs
+    was investigated. In a control study, 60 subjects were divided into
+    three groups each respectively receiving daily doses of 0.5, 1.0 and 2.0
+    mg of vitamin C. After 42 days, the tooth
```

Comment:
Without identifying the groups, I'm not sure that the word "respectively" would be appropriate. In any case, I think it should appear at the end of the sentence.
```suggestion
    small dose, medium dose, and large dose groups that received
    daily doses of 0.5, 1.0 and 2.0 mg of vitamin C, respectively. 
    After 42 days, the tooth
```